helm-charts/helm3/Makefile

```diff
@@ -48,6 +49,12 @@ helm_install: helm_clean helm_template
 	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*ClusterRole-*.yaml' -printf "%p " -exec yq r {} kind \; \
 	| $(SED) -E 's/([^ ]*) ([a-zA-Z0-9]*)$$/\1/' \
 	| xargs -IFILE $(CP) FILE $(CHART_RENDERED_TEMPLATES_CLUSTERROLES)
+	# Copying rendered template files to: $(CHART_RENDERED_TEMPLATES_CONFIGMAPS)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_CONFIGMAPS)
+	# Find rendered resources which are ConfigMaps and move them
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*ConfigMap-*.yaml' -printf "%p " -exec yq r {} kind \; \
+	| $(SED) -E 's/([^ ]*) ([a-zA-Z0-9]*)$$/\1/' \
+	| xargs -IFILE $(CP) FILE $(CHART_RENDERED_TEMPLATES_CONFIGMAPS)
```

Comment:
Could you explain what does this do? It is not really clear to me.

Reply:
It copies rendered ConfigMap files into `cluster-operator/src/main/resources/config-maps/`

Reply:
But why?